src/main/java/com/mrdb/model/GetAndSetData.java

```diff
@@ -0,0 +1,70 @@
+package com.mrdb.model;
+
+import com.mrdb.entities.UserEntity;
+import com.mrdb.entities.UserEntityRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GetAndSetData {
+    private UserEntityRepository repo;
+    public GetAndSetData(UserEntityRepository repo){
+        this.repo = repo;
+    }
+
+    public String saveToDb(UserEntity userDetails) {
+        String result = "Success";
+        try {
+            if(checkEmail(userDetails)) {
```

Comment:
Code is written once but read several times. So, it should be very intuitive to the reader and naming should be descriptive.
When I read `checkEmail`, I expected the method would check if email is empty or if it is a valid email address. However, the implementation is basically looking up if this email already exists in the database. There is a disconnect there.
Hence, I think `doesEmailExist` or `isKnownEmail` is more descriptive of this method rather than `checkEmail`.
Same with the `checkUser` method.